start.sh

```diff
@@ -2,12 +2,8 @@
 
 if [ ! -d MapBot ]; then
     git clone https://github.com/vishakha-lall/MapBot.git -b gssoc-master
-    cd MapBot
-else
-    cd MapBot
-    git pull
```

Comment:
How will git pull work now?

Reply:
Each time `docker-entrypoint.sh` is executed, `git pull` is executed inside the working directory, due to [this line](https://github.com/vishakha-lall/MapBot/pull/107/commits/0fbedb712e88740ffa18030d58ac35ff9293c633#diff-60713fb19f6c50a2473835df63ff17f9R3), I believe. That's how it'll pull the latest code.

Reply:
On hindsight, we can also use `git -C`, for this to remain in `start.sh`, like described [here](https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt) in the documentation.